net-analyzer/nagios-plugins-ceph/nagios-plugins-ceph-1.5.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1
+
+DESCRIPTION="Nagios plugins for Ceph"
+HOMEPAGE="https://github.com/ceph/ceph-nagios-plugins"
+SRC_URI="https://github.com/ceph/ceph-nagios-plugins/archive/ceph-nagios-plugins_${PV}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="
+	acct-group/nagios
+	acct-user/nagios
+"
+RDEPEND="
+	${DEPEND}
+	${PYTHON_DEPS}
+	sys-cluster/ceph
+"
+BDEPEND=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+S="${WORKDIR}/ceph-nagios-plugins-ceph-nagios-plugins_${PV}"
+
+src_prepare() {
+	default
+	python_fix_shebang "${S}"/src/*
+}
+
+src_install() {
+	emake DESTDIR="${D}" install sysconfdir=/etc libdir=/usr/$(get_libdir)
+
+	fowners -R nagios:nagios /usr/$(get_libdir)/nagios/plugins
```

Comment:
Looking at https://github.com/ceph/ceph-nagios-plugins/blob/master/Makefile#L51, I think the default permissions would be OK here? There's no harm in allowing regular users to execute the plugins and it's nice sometimes while debugging a failed check.